airflow/ti_deps/deps/not_previously_skipped_dep.py

```diff
@@ -49,46 +49,45 @@ def _get_dep_statuses(self, ti, session, dep_context):
         finished_task_ids = {t.task_id for t in finished_tis}
 
         for parent in upstream:
-            if isinstance(parent, SkipMixin):
```

Comment:
Initially I though of making it:
`if isinstance(parent, (SkipMixin, MappedOpeartor))`

But then I saw that no test fails without it - so I decided going with Occam's razor and simply remove it.